src/components/index/HomePageHeader.tsx

```diff
@@ -2,12 +2,23 @@ import { Container } from "@components/commons/Container";
 import { useTranslation } from "next-i18next";
 import { Video } from "@components/commons/Video";
 import classNames from "classnames";
+import { useRouter } from "next/router";
 
 export function HomePageHeader(): JSX.Element {
   const { t } = useTranslation("page-index");
+  const router = useRouter();
+
   const separatedTitle = t("Header.title").split(" ");
-  const firstWord = separatedTitle[0];
-  separatedTitle.shift();
+
+  function displayColorToLanguage(index: number): boolean {
+    if (router.locale === "de") {
+      return index === 1;
+    }
+    if (router.locale === "en-US") {
+      return index === 0;
+    }
```

Comment:
changing color by index seems troublesome to maintain. Can you try if this could be improved? 
Perhaps using <Trans> component if i18next package? I haven't used this before, not sure if it's a good idea, if it makes it more complicated, let's stick to this original `index` approach

Reply:
Ive looked at the documentation, it looks like its just another format (but more complicated) of the current simpler approach - lets stick to the existing one instead